adala/runtimes/_litellm.py

```diff
@@ -144,25 +55,15 @@ def init_runtime(self) -> 'Runtime':
             )
         return self
 
-    def execute(self, messages: List):
-        """
-        Execute LiteLLM request given list of messages in OpenAI API format
-        """
-        if self.verbose:
-            print(f'LiteLLM request: {messages}')
-
-        completion = litellm.completion(
+    def get_llm_response(self, messages: List[Dict[str, str]]) -> Dict[str, Any]:
+        response = get_llm_response(
+            messages=messages,
             model=self.model,
             api_key=self.api_key,
-            messages=messages,
             max_tokens=self.max_tokens,
-            temperature=self.temperature,
+            temperature=self.temperature
         )
-        completion_text = completion.choices[0].message.content
-
-        if self.verbose:
-            print(f'LiteLLM response: {completion_text}')
-        return completion_text
+        return response['data']['_completion_text']
```

Comment:
error handling - is the invariant from LSEHandler preserved here that `data == None` iff `adala_error == True`? Can `_completion_text` ever not exist in `data`?

Reply:
resolved it with https://github.com/HumanSignal/Adala/pull/162#discussion_r1692436952